spec/person_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../person'
+
+describe 'Person' do
+  context 'create an instance of person class' do
+    person = Person.new(32, 'Rafi', true)
+
+    it 'returns a Person object from three parameters' do
+      expect(person.class) == Person
+    end
+    it 'returns correct name of person' do
+      expect(person.age).to eq 32
+      expect(person.name).to eq 'Rafi'
+      expect(person.parent_permission).to eq true
+    end
+  end
+
+  context 'create an instance of Person class' do
+    frances = Person.new(32, 'Frances', false)
+    it 'Correct name' do
+      expect(frances.correct_name).to eq 'Frances'
+    end
+
```

Comment:
- [x] Good Job your tests are passing 😉 

![image](https://user-images.githubusercontent.com/75126481/236280927-dd422486-a400-43e6-92b5-9d1aa31606d5.png)

Reply:
Thanks very much @hiromataba for the review. 😁